scripts/create-devices-inventory-from-cvp.py

```diff
@@ -81,23 +81,30 @@ def main():
 
     if args.container is None:
         # Get a list of all devices
-        # help(clnt.api.get_inventory)
         inventory = clnt.api.get_inventory()
-        # write all IP addresses from the CVP inventory in the all.txt file
-        out_file = f'{out_dir}/all.txt'
+        # write the IP addresses in a file
+        out_file = f'{out_dir}/all.yml'
         with open(out_file, 'w', encoding="utf8") as out_file:
+            out_file.write("anta_inventory:")
+            out_file.write('\n')
+            out_file.write("  hosts:")
+            out_file.write('\n')
             for dev in inventory:
```

Comment:
Would be nice to use model we have for inventory to capture CVP data and then save it instead of hardcoding inventory syntax.

Below is a pseudo code that could help:

```python
cvp_data[anta.inventory.AntaInventory.INVENTORY_ROOT_KEY]['hosts'] = []
for dev in container_inventory:
    cvp_data[anta.inventory.AntaInventory.INVENTORY_ROOT_KEY]['hosts'].append(dev['ipAddress'])

with open(cvp_inventory, 'w') as f:
    yaml.dump(cvp_data, f)
```

Reply:
I will use the model from the inventory module

Reply:
will looks like this
```
from anta.inventory import AntaInventory
import yaml

inv={AntaInventory.INVENTORY_ROOT_KEY:{"hosts":[]}}

cvp=["1.1.1.1", '2.2.2.22']

for dev in cvp: 
   inv[AntaInventory.INVENTORY_ROOT_KEY]['hosts'].append({"host":dev})

with open("inventory.yml","w", encoding="utf8") as outfile 
    outfile.write(yaml.dump(inv))
```